src/components/users/user-signup/ko/runtime/user-signup.ts

```diff
@@ -84,10 +84,9 @@ export class UserSignup {
         }
 
         const settings = {
```

Comment:
A couple of things here:
1. We need to take into account that terms of use may not be specified at all.
2. Settings `userRegistrationTermsEnabled` and `userRegistrationTermsConsentRequired` should also come from configuration.

Reply:
k~, will fix that